src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -1,32 +1,10 @@
 package core.basesyntax.dao.impl;
 
-import core.basesyntax.dao.CommentDao;
 import core.basesyntax.model.Comment;
-import java.util.List;
 import org.hibernate.SessionFactory;
 
-public class CommentDaoImpl extends AbstractDao implements CommentDao {
+public class CommentDaoImpl extends AbstractDao<Comment> {
```

Comment:
```suggestion
public class CommentDaoImpl extends AbstractDao<Comment> implements CommentDao {
```
the same with other implementations